### PR TITLE
docs: add documentation to setup() arguments

### PR DESCRIPTION
**Pull request by martinm82**

### CODE REVIEW

File: {{cookiecutter.project_name}}/setup.cfg [hunk 1]
```diff
@@ -29,7 +29,7 @@ collect_ignore = ['setup.py']
 
 [coverage:report]
 show_missing = true
-fail_under = 65
+fail_under = 70
```
Comment (eugene-davis) [inline {{cookiecutter.project_name}}/setup.cfg]: Why does this increase in this PR?

File: {{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/cli.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 
 def parse_args(args):
-    """Returns parsed commandline arguments.
+    """Returns parsed command line arguments.
```
Comment (eugene-davis) [inline {{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/cli.py]: :+1:

File: {{cookiecutter.project_name}}/setup.cfg (hunk 1 above)
Comment (martinm82) [reply {{cookiecutter.project_name}}/setup.cfg]: Yeah, not needed necessarily here. Same goes for the spellcheck errors

Comment (martinm82) [reply {{cookiecutter.project_name}}/setup.cfg]: Removed it

File: {{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/cli.py (hunk 2 above)
Comment (martinm82) [reply {{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/cli.py]: As discussed off-line. Keeping this change in this PR

**Pull request merged**